spec/support/step_definitions/auth_steps.rb

```diff
@@ -1,11 +1,20 @@
 # frozen_string_literal: true
 
 module AuthSteps
-  step "I am a valid API user with username :username" do |username|
+  step "I am a :content_type collection manager with username :username" do |content_type, username|
     key = Keycard::DigestKey.new
     @user = Fabricate(:user, username: username, api_key_digest: key.digest)
     header "Authorization", "Token token=#{key}"
+
+    Services.checkpoint.grant!(@user,
+                               Checkpoint::Credential::Role.new('content_manager'),
+                               Checkpoint::Resource::AllOfType.new(content_type))
   end
 end
 
-RSpec.configure {|config| config.include AuthSteps }
+RSpec.configure do |config|
+  config.include AuthSteps
+  config.after(:each, type: :feature) do
+    Checkpoint::DB.db[:grants].delete
+  end
```

Comment:
It looks like we can require `checkpoint_helper` here and skip this part if we do one of three things:

1. Use a separate, additional tag for transactionality
2. Set up a loop for an `around` for each type
3. Set up a generic `around` that conditionally wraps in a transaction based on `example.metadata[:type]`

I think I'd probably prefer the first, and go with something like `checkpoint: :transactional` to be totally explicit.

Reply:
Uses a `:checkpoint_transaction` tag, since turnip/cucumber/gherkin can't expose the `checkpoint: : transactional` form.